packages/dialog/src/DialogWrapper.ts

```diff
@@ -125,6 +125,7 @@ export class DialogWrapper extends DialogBase {
         }
 
         return html`
+            ${this.dismissHelper}
```

Comment:
The first dismissHelper is always shown, regardless of `dismissable`. 

The one at the end of this render function is conditional, so that at most a dialog would only have 2 options for dismissal- a visible close button, and a visually-hidden one for screen readers. 

Is that still too redundant for screen readers users when `dismissable` is true and there's a rendered `sp-close-button`?